doc/user/ospf6d.rst

```diff
@@ -170,10 +170,12 @@ Showing OSPF6 information
    instance ID, simply type "show ipv6 ospf6 <cr>". JSON output can be
    obtained by appending 'json' to the end of command.
 
-.. index:: show ipv6 ospf6 database
-.. clicmd:: show ipv6 ospf6 database
+.. index:: show ipv6 ospf6 database [json]
```

Comment:
Let's take the time and actually update the documentation in a way that actually reflects the changes you are making plase.

Reply:
Updated